tox.ini

```diff
@@ -1,26 +1,22 @@
 [tox]
-envlist =
-    py27-nameko{2.9, 2.10, 2.11, 2.12, latest},
-    {py35, py36, py37, py38}-nameko{2.7, 2.8, 2.9, 2.10, 2.11, 2.12, latest, next}
-
+envlist = {py36, py37, py38, py39}-nameko{2.12, 2.13, 2.14}
```

Comment:
Does this library work with newer version of Python (3.10, 3.11)? Is that the reason why we're not including them here?

Reply:
Let's try